### ISSUE TITLE
[ERROR] not in capture list. when adding new models.

### ISSUE THREAD

**Issue by PurrSevere** [jrudess/streamdvr#111]
Something in yesterday's updates introduced a new bug where I can't add any new models. Whenever I try, I get the error message:

```
1 streams(s) to include
[ERROR] model_name not in capture list.
```

I have only tested this in MFC and CB. I am updating models manually by adding them to the site_updates.yml file.

**Comment by Asynchr0nous** [comment]
i can confirm this is the case with mfc. i wanted to test some more before creating a new issue. i can remove but not add at all.

**Comment by jrudess** [comment]
Confirmed.  Using the TUI to add/remove is working still, and I missed testing the yml.

**Comment by jrudess** [comment]
Checked in a fix for adding.  Deleting is broken if you edit mfc_updates.yml while streamdvr is not running.   There is a start-time load ordering issues, where the updates file is processed first, but the streamerlist for mfc is not yet loaded, so it can't find the streamer and still prints the same error.

**Comment by PurrSevere** [comment]
Can confirm working now.

**Comment by jrudess** [comment]
Ideally adding streamers should occur before starting up the sites, so the first round of fetches looks them up and starts recording.  But deleting a streamer shouldn't occur until after the sites have loaded their lists.   

The deleting may have been broken for a little while.  I don't fully comprehend yet how the recent  changes have affected ordering.

**Comment by jrudess** [comment]
some issues with CB recordings not started, probably related to this changeset

**Comment by jrudess** [comment]
resolved
